### PR TITLE
Support redir for builtins

### PR DESCRIPTION
**Pull request by rpaquay**
@ussuri

### CODE REVIEW

File: lib/wash/exe/wash.js [hunk 1]
```diff
@@ -561,62 +555,84 @@ Wash.prototype.dispatch = function(pathSpec, argv, redirPathSpec, redirMode) {
...
     }
+
+    return Promise.all(cxPromises).then(
+      function(cxs) {
+        var execCx = cxs[0];
+        var redirCx = cxs[1];
+
+        execCx.onClose.addListener(function(reason, value) {
+          if (reason === 'ok') {
+            if (typeof value != 'undefined' && typeof value != 'number' &&
```
Comment (ussuri) [inline lib/wash/exe/wash.js]: Since we're touching this code here, let's fix the comparisons to `!==` instead of `!=`.

Comment (ussuri) [inline lib/wash/exe/wash.js]: Although skipped printing for numbers seems strange: isn't it possible that an executable would legitimately return a number as a result?

File: lib/wash/exe/wash.js [hunk 2]
```diff
@@ -551,7 +545,7 @@ Wash.prototype.dispatch = function(pathSpec, argv, redirPathSpec, redirMode) {
    * @param {!Path} path
    * @return {!Promise<StatResult>}
    */
-  var builtinExists = function(path) {
+  var findBuiltin = function(path) {
```
Comment (ussuri) [inline lib/wash/exe/wash.js]: I would make this function a member of `Wash.prototype`, next to `Wash.prototype.findExecutable()`: more symmetrical and would avoid recreation of the function on every command execution.

This is very very minor, though.

**Comment by ussuri** [comment]
A couple of nits, otherwise LGTM.

File: lib/wash/exe/wash.js (hunk 1 above)
Comment (rpaquay) [inline lib/wash/exe/wash.js]: Comparisons: done.
number test: it came from the existing code in another location.

File: lib/wash/exe/wash.js (hunk 2 above)
Comment (rpaquay) [inline lib/wash/exe/wash.js]: Done.

**Pull request merged**